docs/maven-tutorial.md

```diff
@@ -0,0 +1,150 @@
+# Maven + Nexus Tutorial
+
+While developing your application with Java and Maven, you'll most likely be
+building many times. In order to shorten the build times of your
+pods, Maven dependencies can be cached in a local Nexus repository. This
+tutorial will step you through creating a Nexus repository on your cluster.
+
+|Content:|
+|:--------|
+|[Prerequisites](#prerequisites)|
+|[Setting up Nexus](#setting-up-nexus)|
+|[Connecting to Nexus](#connecting-to-nexus)|
+|[See Also](#see-also)|
+
+## Prerequisites
+
+This tutorial assumes that you'll be working with a project that is already set
+up for use with Maven. If you're interested in using Maven with your Java
+project, we highly recommend looking at
+[their guide](https://maven.apache.org/guides/getting-started/index.html).
+
+In addition, be sure to check your application's image for Maven mirror
+capabilities. Many images that use Maven have a 'MAVEN_MIRROR_URL' environment
+variable that you can use to simplify this process. If it doesn't have this
+capability, read
+[the Nexus Maven documentation](https://books.sonatype.com/nexus-book/reference/config.html)
+to configure your build properly.
+
+## Setting up Nexus
+
+Use `new-app` to download and deploy the official Nexus Docker image.
+```bash
+oc new-app sonatype/nexus
+```
+
+After the build has finished, run:
+```bash
+oc get svc
+```
+
+The output should resemble:
+```bash
+NAME      CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
+nexus     172.30.170.249   <none>        8081/TCP   6s
+```
+
+Confirm that Nexus is running by navigating your browser to the new Cluster-IP
+at port 8081. The server may take a bit of time to start, be patient!
+
+> NOTE: Nexus comes pre-configured for the Central Repository, but you may need
+> others for your application. At this point, add any special repositories to
+> Nexus. Documentation on how to add resositories can be found
+> [here](https://books.sonatype.com/nexus-book/reference/config-sect-new-repo.html).
```

Comment:
is this doc going to be sufficient someone to be able to update the nexus repo they've just deployed?  (does it require updating the image?  logging into the repo in which case they need the service ip or a route?)